src/components/Task.vue

```diff
@@ -56,12 +65,31 @@ export default defineComponent({
     NTag,
     TreeViewAlt
   },
+  methods:{
+    getAssigneeInitials(assignee: string) {
+      if (assignee) {
+        const nameSplit = assignee.split(" ");  
```

Comment:
What if a name doesn't have a space, will this all work?

Reply:
Yes, nameSplit would then be an array of 1 string, and only the first initial will be displayed in the icon.

Reply:
👍